config/checkstyle/sun_checks.xml

```diff
@@ -204,4 +204,12 @@
     <property name="severity" value="warning"/>
   </module>
 
+  <!-- Checks for Headers to not contain old license text -->
+  <module name="RegexpSingleline">
+    <property name="format" value="Portions Copyright OpenSearch Contributors"/>
```

Comment:
For now I'm checking the starting line of the old license header to see if any files contain it, if so checkstyle will throw violation.